README.md

```
@@ -232,6 +232,43 @@ curl http://localhost:8080/health
+
+`config.yaml`で以下のように設定：
+
+```yaml
+database:
+  dev: "./data/upgo-dev.db" # 開発環境用
+  prd: "./data/upgo.db" # 本番環境用
+```
+
+#### 環境の切り替え
+
+**開発環境（デフォルト）**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)